Use GitHub Actions for CI.

pwalsh opened the pull request:
Use GitHub Actions for CI.

pwalsh commented:
@akariv I have no idea why coveralls check hangs. I reset the coveralls settings with GitHub. I'm wondering if coveralls is broken somehow as last run was Sept 2020 but there has been activity in the repo since then.

https://coveralls.io/github/frictionlessdata/tableschema-sql-py

Can you take a quick look?

akariv commented:
I just removed coveralls from ts-es as well. We have codecov here which seems to be a good replacement (?)

pwalsh commented:
> I just removed coveralls from ts-es as well. We have codecov here which seems to be a good replacement (?)

@akariv ok, I deleted the integration in coveralls.
